CRM/Civicpd/Page/CPDReport.php

```diff
@@ -1476,68 +1476,51 @@ function civi_cpd_report_update_activity()
 
 
     if (!empty($_POST['activity_id']) && !empty($_POST['category_id']) && civi_cpd_report_validate_date($_POST['credit_date']) && civi_cpd_report_validate_number($_POST['credits']) && !empty($_POST['notes']) && !empty($_POST['activity'])) {
-
-        $activityId = $_POST['activity_id'];
-
-        $startDate = (isset($_POST['start_date'])
-            && civi_cpd_report_validate_start_date($_POST['start_date']) ) ? "start_date='".$_POST['start_date']."'," : "start_date=NULL,";
-
-        $creditDate = $_POST['credit_date'];
-
-        $credits = number_format($_POST['credits'], 2, '.', '');
-
-        $activity = $_POST['activity'];
-
-        $notes = $_POST['notes'];
-
         $evidence = civi_cpd_report_import_activity_evidence_pdf();
-
-
         if ($evidence === false) {
-
             CRM_Civicpd_Page_CPDReport::redirectToReport();
-
         }
 
+        $updateCpdActivityQuery = "UPDATE civi_cpd_activities SET 
+                                   start_date = %1,
+                                   credit_date = %2,
+                                   credits = %3,
+                                   activity = %4,
+                                   notes = %5,
+                                   evidence = %6 
+                                   WHERE id = %7";
+
+        $startDate = NULL;
```

Comment:
Have you tested with `$startDate = NULL`? Just it seems a little different from the old implementation where the string `SET start_date = NULL` now changes to using `'start_date', NULL` in the params. If you've tested it it's fine, just thought I'd check.

Reply:
well, actually I told you to review and I were installing the extension to test these new changes and this case specifically